File: Anya_Seth/Sliding Window/Maximum Points You Can Obtain from Cards.java
class Solution {
    public int maxScore(int[] cardPoints, int k) {
        int i = k - 1;
        int j = cardPoints.length - 1;
        int score = 0;
        for(int z = 0 ; z < k ; z++){
            score += cardPoints[z];
        }
        int maxScore = score;
        while(i >= 0){
            score += cardPoints[j];
            score -= cardPoints[i];
            maxScore = Math.max(maxScore, score);
            i--;
            j--;
        }
        return maxScore;
    }
}

File: Lakshay Garg/Leetcode/Sliding Window/Maximum Points You Can Obtain from Cards.cpp
class Solution {
public:
    int maxScore(vector<int>& v, int k) {
        int n=v.size(),ans=0;
        for(int i=0;i<k;i++) ans+=v[i];
        int count=ans;
        for(int i=k-1;i>=0;i--){
            count-=v[i];
            count+=v[n-k+i];
            ans = max(ans,count);
        }
        return ans;
    }
};
